src/restic/node.go

```diff
@@ -14,10 +14,17 @@ import (
 
 	"runtime"
 
+	"bytes"
 	"restic/debug"
 	"restic/fs"
 )
 
+// Xattr is a tuple storing the xattr name and value.
+type Xattr struct {
+	XattrName  string `json:"xattrname"`
```

Comment:
No need to repeat yourself, use `Name` (`json:"name"`) and `Value` (`json:"value")`.

This is a lot easier to read in the code:
```go
attr.XattrnNme := "foo"
```
versus
```go
attr.Name := foo
```